jdkfuncs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# This code taken from:
+# https://www.jayway.com/2014/01/15/how-to-switch-jdk-version-on-mac-os-x-maverick
+# It allows setting the JDK version to use in the current process.
+function setjdk() {
```

Comment:
Oh nice... that would work better for portability. I'll take a look! Thanks for the recommendation!